package.json

```diff
@@ -136,7 +135,7 @@
     "build-docs-watch": "gulp buildDocsWatch",
     "eslint": "eslint \"./**/*.js\" \"./**/*.cjs\" \"./**/*.html\" --cache --quiet",
     "make-zip": "gulp makeZip",
-    "markdownlint": "markdownlint \"*.md\" \"Documentation/**/*.md\" \"packages/**/*.md\" --ignore CHANGES.md --ignore \"./**/LICENSE.md\"",
```

Comment:
I like the .markdownlintignore file for CHANGES and LICENSE, etc.--it is a cleaner approach.
But currently markdownlint is checking files like `Documentation/CustomShaderGuide/README.md`. With this change, those files will be ignored. Was that the intent here?

Reply:
My mistake; I thought that was an ignored directory.